lib/arena.rb

```diff
@@ -1,2 +1,29 @@
 class Arena
+  attr_accessor :name, :gladiators
+
+
+    def initialize(name)
+      @name = name.capitalize
+      @gladiators = []
+    end
+
+    def add_gladiator(gladiator)
+      if @gladiators.length < 2
+        @gladiators.push(gladiator)
+      else
```

Comment:
You don't always need an `else`. You can have an `if` by itself (but not an `elsif` or an `else` by itself).